src/freenet/node/simulator/RealNodeRequestInsertTest.java

```diff
@@ -172,8 +176,8 @@ int insertRequestTest() throws CHKEncodeException, InvalidCompressionCodecExcept
         Node randomNode = nodes[node1];
         //Logger.error(RealNodeRequestInsertTest.class,"Inserting: \""+dataString+"\" to "+node1);
         
-        //boolean isSSK = requestNumber % 2 == 1;
-        boolean isSSK = true;
+        boolean isSSK = requestNumber % 2 == 1;
+        //boolean isSSK = true;
```

Comment:
@toad could you add a note to the commented out code when it’s useful in testing? Aside from that I think it’s OK to keep the entry point for quick testing. All cleaner ways I see add multiple lines of overhead.